cypress.js

```diff
@@ -0,0 +1,10 @@
+const fs = require('fs-extra');
```

Comment:
See https://docs.cypress.io/guides/guides/module-api#cypress-cli

This allows running Cypress with a different `cwd`